.changeset/beige-chefs-confess.md

```diff
@@ -0,0 +1,40 @@
+---
+"@comet/eslint-config": major
+---
+
+Enforce PascalCase for enums
+
+Changing the casing of an existing enum can be problematic, e.g. if the enum values are persisted in the database. 
+In such cases, the rule can be disabled like so
+
+```diff
++ /* eslint-disable @typescript-eslint/naming-convention */
+  export enum ExampleEnum {
+      attr1 = "attr1",
+  }
++ /* eslint-enable @typescript-eslint/naming-convention */
+```
+
+You can also "disable" the rule in your whole project by overriding it and allowing multiple casings:
+
+```json 
+// .eslintrc.json
+"overrides": [
+    {
+        "files": ["*.ts", "*.tsx"],
+        "rules": {
+            "@typescript-eslint/naming-convention": [
+                "error",
+                {
+                    "selector": "enum",
+                    "format": ["PascalCase", "camelCase", "UPPER_CASE"]
+                },
+                {
+                    "selector": "enumMember",
+                    "format": ["PascalCase", "camelCase", "UPPER_CASE"]
+                }
+            ]
+        }
+    }
+]
+```
```

Comment:
Sadly it's not possible to disable parts of `@typescript-eslint/naming-convention`. Overrides is one option suggested by one of the maintainers (https://github.com/typescript-eslint/typescript-eslint/issues/2755#issuecomment-725708267)

Reply:
I'd remove this suggestion from the changeset. We want our developers to adhere to our naming conventions as much as possible. If a developer _really_ wants to disable it for the whole project (which I'd strongly advise against), they will find the GitHub issue themselves.

Reply:
Done bf5b62e3cfb47b387623dab38d6420ab1893e687